src/zope/container/_zope_container_contained.c

```diff
@@ -65,19 +72,17 @@ typedef struct {
 #define OBJECT(O) ((PyObject*)(O))
 #define Proxy_GET_OBJECT(ob)   (((ProxyObject *)(ob))->proxy_object)
 
-#define CLEAR(O) \
-  if (O) {PyObject *clr__tmp = O; O = NULL; Py_DECREF(clr__tmp); }
+#define CLEAR(O) Py_CLEAR(O)
 
-/* Supress inclusion of the original proxy.h */
-#define _proxy_H_ 1
-
-/* Incude the proxy C source */
+/* Incude the proxy  source */
```

Comment:
Typo: Incude -> Include.  Also the double spaces remaining after the removal of `C` seem strange?

Reply:
Whoops. Thanks. That's a result of some earlier experiments that didn't fully get rolled back (I'd really like to avoid having to embed the .c file, but figuring out what that would entail is beyond me at the moment).